src/Goerli-DssSpell.t.sol

```diff
@@ -103,36 +99,44 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         checkCollateralValues(afterSpell);
     }
 
-    function testRemoveChainlogValues() private { // make public to use
+    function testRemoveChainlogValues() public { // make public to use
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // try chainLog.getAddress("XXX") {
-        //     assertTrue(false);
-        // } catch Error(string memory errmsg) {
-        //     assertTrue(cmpStr(errmsg, "dss-chain-log/invalid-key"));
-        // } catch {
-        //     assertTrue(false);
-        // }
+        try chainLog.getAddress("RWA007_A_INPUT_CONDUIT_URN") {
+            assertTrue(false);
+        } catch Error(string memory errmsg) {
+            assertTrue(cmpStr(errmsg, "dss-chain-log/invalid-key"));
+        } catch {
+            assertTrue(false);
+        }
+
+        try chainLog.getAddress("RWA007_A_INPUT_CONDUIT_JAR") {
+            assertTrue(false);
+        } catch Error(string memory errmsg) {
+            assertTrue(cmpStr(errmsg, "dss-chain-log/invalid-key"));
+        } catch {
+            assertTrue(false);
+        }
 
     }
 
-    function testCollateralIntegrations() private { // make public to use
+    function testCollateralIntegrations() public { // make public to use
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
         // Insert new collateral tests here
-        // checkIlkIntegration(
-        //     "RETH-A",
-        //     GemJoinAbstract(addr.addr("MCD_JOIN_RETH_A")),
-        //     ClipAbstract(addr.addr("MCD_CLIP_RETH_A")),
-        //     addr.addr("PIP_RETH"),
-        //     true, /* _isOSM */
-        //     true, /* _checkLiquidations */
-        //     false /* _transferFee */
-        // );
+        checkIlkIntegration(
```

Comment:
please enable also `testIlkClipper` for reth.

Reply:
Have some problems with this test, I am getting `Dog/not-unsafe` when on `.bark()` call.